core/gethwrappers/generation/generate/genwrapper/genwrapper.go

```diff
@@ -5,8 +5,6 @@ import (
 	"os"
 	"path/filepath"
 
-	gethParams "github.com/ethereum/go-ethereum/params"
-
 	gethwrappers2 "github.com/smartcontractkit/chainlink/v2/core/gethwrappers"
```

Comment:
```suggestion
	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers"
```
I feel we might not need to rename here